py3status/modules/xrandr.py

```diff
@@ -227,9 +229,7 @@ def _apply(self, force=False):
                 if mode == 'clone' and previous_output is not None:
                     cmd += ' --auto --same-as {}'.format(previous_output)
                 else:
-                    if 'left-of' in pos:
-                        cmd += ' --auto --{} --rotate normal'.format(pos)
-                    elif 'right-of' in pos:
+                    if 'left-of' in pos or 'right-of' in pos or 'above' in pos or 'below' in pos:
```

Comment:
would you mind sorting the if alphabetically ? (above, below, left, right) and making sure it does not exceed 80 chars on the line ?